.github/labels.yml

```diff
@@ -0,0 +1,277 @@
+  name: stale
+- color: bfdadc
+  description: ""
+  name: WIP
+
+# ##### status labels ##### #
+- color: cfd3d7 # gray (standard github)
+  description: This issue or pull request already exists
+  name: duplicate # status:duplicate
+- color: e4e669 # yellow (standard github)
+  description: This doesn't seem right
+  name: invalid # status:invalid
+- color: ffffff # white (standard github)
+  description: This will not be worked on
+  name: wontfix # status:wontfix
+- color: C0C0C0
+  description: ""
+  name: status:can't reproduce
+- color: 7b5804
+  description: ""
+  name: status:need more info
+- color: E99695
+  description: ""
+  name: status:needs changes
+- color: D4C5F9
+  description: A manual tag to note PRs that are otherwise ready, but the CLA bot says the CLA has not been signed.
+  name: status:needs cla
+- color: d93f0b
+  description: ""
+  name: status:needs tests
+- color: cccccc
+  description: ""
+  name: status:not an issue
+- color: cccccc
+  description: Feature, design or idea with undecided status to include in the project functionality
+  name: status:on hold
+- color: fbca04
+  description: ""
+  name: status:to be verified
+- color: f7c6c7
+  description: ""
+  name: status:under discussion
+- color: 8FFCDE
+  name: "status: potential performance implications"
+  description: ""
+
+# ##### area/misc labels ##### #
+- color: FFFF5F
+  description: ""
+  name: external dependency
+- color: 2C5B58
+  description: ""
+  name: 'infrastructure: ci/cd'
+- color: 5F5F5F
+  description: ""
+  name: maintenance
+- color: d7e102
+  description: ""
+  name: tests
+
+# ##### area/code_location labels ##### #
+- color: 0075ca # blue (standard github)
+  description: Improvements or additions to documentation
+  name: documentation
+- color: 0075ca # blue
+  description: ""
+  name: API
+- color: 0075ca # blue
+  description: ""
+  name: API/v2
+- color: 0075ca # blue
+  description: ""
+  name: CLI
+- color: 0075ca # blue
+  description: ""
+  name: component:st2auth
+- color: 0075ca # blue
+  description: ""
+  name: component:st2client
+- color: 0075ca # blue
+  description: ""
+  name: component:st2web
+- color: 0075ca # blue
+  description: ""
+  name: migrations
+- color: 0075ca # blue
+  description: ""
+  name: runners
+- color: 0075ca # blue
+  description: ""
+  name: 'service: action runner'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: api'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: auth api'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: rules engine'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: scheduler'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: workflow engine'
+
+# ##### area/python labels ##### #
+- color: 0075ca # blue
+  description: ""
+  name: python3
+- color: 0075ca # blue
+  description: ""
+  name: unicode
+
+# ##### area/pack_resources labels ##### #
+- color: 003459 # blue
+  description: ""
+  name: action aliases
```

Comment:
I think we can use this color of blue for `area/pack_resources` labels. Here is how `#003459` looks in dark and light mode:
![image](https://user-images.githubusercontent.com/1558590/144345269-c7c5a938-d213-487a-a2cd-83553835ee3e.png)